Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java

```diff
@@ -128,13 +171,36 @@ public void getTableAsString() {
     }
   }
 
-  public void insertScriptedDataWithCheckDuplication(String url, String title, String content, int expectedTime, String imgUrl) {
-    SQLiteDatabase dataBase = getReadableDatabase();
-    Cursor cursor = dataBase.rawQuery("SELECT * FROM PAGE WHERE URL = ('" + url + "') ", null);
+  /**
+   * Please Check URL duplication before using insert method.
+   * Using Duplication()
+   * @param url
+   */
+  public void insertScriptedData(String url, String title, String content, int expectedTime, String imgUrl) {
+    SQLiteDatabase dataBase = getWritableDatabase();
+    String escapedTitle = StringEscapeUtils.escapeHtml4(title);
+    String escapedContent = StringEscapeUtils.escapeHtml4(content);
+    dataBase.execSQL("INSERT INTO PAGE (URL, TITLE, CONTENT, EXPECTEDTIME, IMGURL) VALUES ('" + url + "',\"" + escapedTitle + "\", \"" + escapedContent + "\", " + String.valueOf((int)expectedTime) + " , '" + imgUrl + "');");
+    dataBase.close();
```

Comment:
Not sure but. 

try (SQLiteDatabase dataBase = getWriteableDatabase()) {
   xxx. 
}

You don't need to call "close()"